src/lib.ts

```diff
@@ -212,199 +204,13 @@ export function tryFeast(familiar: Familiar): void {
-  ),
-
-  new FreeRun(
-    "Middle Finger",
-    () => have($item`mafia middle finger ring`) && !get("_mafiaMiddleFingerRingUsed"),
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).skill(
-      $skill`Show them your ring`
-    ),
-    new Requirement([], { forceEquip: $items`mafia middle finger ring` })
-  ),
-
-  new FreeRun(
-    "VMask",
-    () => have($item`V for Vivala mask`) && !get("_vmaskBanisherUsed"),
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).skill($skill`Creepy Grin`),
-    new Requirement([], { forceEquip: $items`V for Vivala mask` }),
-    () => restoreMp(30)
-  ),
-
-  new FreeRun(
-    "Stinkeye",
-    () =>
-      getFoldGroup($item`stinky cheese diaper`).some((item) => have(item)) &&
-      !get("_stinkyCheeseBanisherUsed"),
-
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).skill(
-      "Give Your Opponent the Stinkeye"
-    ),
-    new Requirement([], { forceEquip: $items`stinky cheese eye` }),
-    () => {
-      if (!have($item`stinky cheese eye`)) cliExecute(`fold stinky cheese eye`);
-    }
-  ),
-
-  new FreeRun(
-    "Navel Ring",
-    () => have($item`navel ring of navel gazing`) && get("_navelRunaways") < 3,
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).step("runaway"),
-    new Requirement([], { forceEquip: $items`navel ring of navel gazing` })
-  ),
-
-  new FreeRun(
-    "GAP",
-    () => have($item`Greatest American Pants`) && get("_navelRunaways") < 3,
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).step("runaway"),
-    new Requirement([], { forceEquip: $items`Greatest American Pants` })
-  ),
-
-  new FreeRun(
-    "Scrapbook",
-    () => {
-      visitUrl("desc_item.php?whichitem=463063785");
-      return have($item`familiar scrapbook`) && get("scrapbookCharges") >= 100;
-    },
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).skill(
-      "Show Your Boring Familiar Pictures"
-    ),
-    new Requirement([], { forceEquip: $items`familiar scrapbook` })
-  ),
-
-  new FreeRun(
-    "Parasol",
-    () =>
-      have($item`peppermint parasol`) &&
-      globalOptions.ascending &&
-      get("parasolUsed") < 9 &&
-      get("_navelRunaways") < 3,
-    Macro.trySkill($skill`Asdon Martin: Spring-Loaded Front Bumper`).item($item`peppermint parasol`)
-  ),
-];
-
-export function findRun(useFamiliar = true): FreeRun | undefined {
-  return freeRuns.find(
-    (run) => run.available() && (useFamiliar || !["Bander", "Boots"].includes(run.name))
-  );
-}
-
-export const ltbRun: () => FreeRun = () => {
-  const freeRunItem = $items`Louder Than Bomb, divine champagne popper, tennis ball`.sort(
-    (a, b) => mallPrice(a) - mallPrice(b)
-  )[0];
-  return new FreeRun(
-    "LTB",
-    () => retrieveItem(freeRunItem),
-    Macro.item(freeRunItem),
-    new Requirement([], {}),
-    () => retrieveItem(freeRunItem)
-  );
+export const ltbRun: () => ActionSource = () => {
```

Comment:
It feels like the new ActionSource system obsoletes this, but also I get that we probably don't want to do too much surgery on the grape all at once

Reply:
yes that's why i left it, i'm gonna do another pass